site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -885,11 +885,12 @@ components:
                required: false
                description: |
                   Specifies the template to use by the device registry to generate the `device-id` during 
-                  auto-provisioning of devices and gateways. The placeholders `{{subject-dn}}` and `{{subject-cn}}` 
-                  are currently supported. The template MUST contain at least one of these placeholders. The `device-id`
-                  is generated by the device registry by replacing any placeholders in the template string with their
-                  corresponding *Subject DN* or *Common Name* from the client certificate used for authentication by 
-                  unregistered devices and gateways. The *Subject DN* is formatted according to the rules defined in 
+                  auto-provisioning of devices and gateways. The placeholders `{{subject-cn}}`, `{{subject-dn}}`,
+                  `{{subject-ou}}` and `{{subject-o}}` are currently supported. The template MUST contain at least
+                  one of these placeholders. The `device-id` is generated by replacing any placeholders in the template
+                  string with their corresponding *Common Name (CN)*, *Subject DN (DN)*, *Organizational Unit Name (OU)*
```

Comment:
IMHO we should make it more explicit, that the CN, OU and O are actually taken from the certificate's subject DN (instead of the issuer DN)